doc/using-logging.rst

```diff
@@ -7,26 +7,33 @@ framework. ZConfig provides one simple convenience function to do this:
 
 .. autofunction:: ZConfig.configureLoggers
 
-
 Suppose we have the following logging configuration in a file called ``simple-root-config.conf``:
 
 .. literalinclude:: simple-root-config.conf
```

Comment:
That seems like a reasonable approach too.

I wasn't aware of this.  I assume this shows up on RTD.

Cool.